From 238e9b80ed62c55eb860565415661c8bf9e6338e Mon Sep 17 00:00:00 2001
From: Sean Mackrory <sean@cloudera.com>
Date: Thu, 19 Jan 2017 08:38:02 -0700
Subject: [PATCH 2228/2748] CLOUDERA-BUILD. Use the same versioning and
 shading for DynamoDB as the rest of the AWS SDK.

Change-Id: I7005cd63e27415491186f1cfe7ef617aa529d8f6
---
 hadoop-project/pom.xml          |    2 +-
 hadoop-tools/hadoop-aws/pom.xml |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index a127bdd..53336c2 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -689,7 +689,7 @@
       <dependency>
         <groupId>com.amazonaws</groupId>
         <artifactId>DynamoDBLocal</artifactId>
-        <version>1.11.0</version>
+        <version>${aws-java-sdk.version}</version>
       </dependency>
       <dependency>
         <groupId>com.amazonaws</groupId>
diff --git a/hadoop-tools/hadoop-aws/pom.xml b/hadoop-tools/hadoop-aws/pom.xml
index 0f93a3d..861255e 100644
--- a/hadoop-tools/hadoop-aws/pom.xml
+++ b/hadoop-tools/hadoop-aws/pom.xml
@@ -305,6 +305,7 @@
                 <includes>
                   <include>com.amazonaws:aws-java-sdk-sts:jar:*</include>
                   <include>com.amazonaws:aws-java-sdk-s3:jar:*</include>
+                  <include>com.amazonaws:aws-java-sdk-dynamodb:jar:*</include>
                   <include>com.amazonaws:aws-java-sdk-core:jar:*</include>
                   <include>com.fasterxml.jackson.*:*</include>
                   <include>joda-time:joda-time:jar:*</include>
-- 
1.7.9.5

